.gitignore

```diff
@@ -7,6 +7,7 @@ db/
 *.dll
 *.so
 *.dylib
+/ziti/cmd/ziti/ziti
```

Comment:
this cannot be intentional - is it?

Reply:
It was intentional. I was trying to ignore the binary created during build but since it doesn't have an extension I can't specify a wildcard of some sort. Unless you have any ideas?

Reply:
the problem is that it all depends on how you choose to build and where you choose to put your built files. i'd say we should probably revert this